### DISCUSSION TITLE
"title must be of type String and cannot be empty", but I would want it empty

### DISCUSSION THREAD

**Discussion by flying-sausages** [community/community#4627] [Projects and Issues] [answered]
I don't need to have something like `[BUG]` at the front of every issue title if I have only one issue form, or prefer to use labels for it.

**Comment by lukehefson** [comment, marked as answer]
Hi @flying-sausages! [As noted in our docs](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms#top-level-syntax)– `title` is an optional top-level syntax so you don’t need to include it at all. You could instead do something like:

```
name: Bug Report
description: File a bug report
body:
…
…
…
```

**Comment by turbolent** [comment]
Ran into this when migrating an existing Markdown issue template.

Please add an exception and treat empty properties (e.g. `title: ''`) the same as not existing, or improve the existing error message to suggest removing the property if it is not needed, as `title: ''` is currently supported in Markdown issue templates.
